data/names.txt

```diff
@@ -3518,6 +3448,161 @@ phrase "pirate"
 		"Fate"
 		"Boys"
 
+#Historical male pirates
```

Comment:
I'd just suggest adding a space after the comment mark for easier reading, here and everywhere else.
```suggestion
# Historical male pirates
```